DHI inline patch (cassandra 4.0): make Guava 32.x's failureaccess explicit

Guava 32.x split failureaccess into a separate runtime jar that was bundled
in Guava 27.0. We bump guava in cassandra-4-mvn-dependencies.json; this
patch removes the now-stale exclusion and adds the explicit dependency.

Upstream Cassandra source: https://github.com/apache/cassandra/blob/cassandra-4.0.19/build.xml
Build wiring: image/cassandra/patch/cassandra-4-mvn-dependencies.json (DHI)

diff --git a/build.xml b/build.xml
index bb5b50e..8d90e2c 100644
--- a/build.xml
+++ b/build.xml
@@ -548,11 +548,11 @@
           <dependency groupId="org.xerial.snappy" artifactId="snappy-java" version="1.1.10.4"/>
           <dependency groupId="at.yawk.lz4" artifactId="lz4-java" version="1.11.1"/>
           <dependency groupId="com.github.luben" artifactId="zstd-jni" version="1.5.7-2"/>
+          <dependency groupId="com.google.guava" artifactId="failureaccess" version="1.0.2"/>
           <dependency groupId="com.google.guava" artifactId="guava" version="32.0.1-jre">
             <exclusion groupId="com.google.code.findbugs" artifactId="jsr305" />
             <exclusion groupId="org.codehaus.mojo" artifactId="animal-sniffer-annotations" />
             <exclusion groupId="com.google.guava" artifactId="listenablefuture" />
-            <exclusion groupId="com.google.guava" artifactId="failureaccess" />
             <exclusion groupId="org.checkerframework" artifactId="checker-qual" />
             <exclusion groupId="com.google.errorprone" artifactId="error_prone_annotations" />
           </dependency>
